tensornetwork/contractors/custom_path_solvers/example.py

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
This file should be moved to `examples/custome_path_solvers`

Reply:
moved the example to a separate folder